exporters/ostream/test/ostream_log_test.cc

```diff
@@ -246,13 +247,14 @@ TEST(OStreamLogExporter, LogWithVariantTypesToClog)
       "  severity_num  : 0\n"
       "  severity_text : INVALID\n"
       "  body          : \n",
-      "  resource      : {",
-      "{service.name: unknown_service}",
-      "{telemetry.sdk.version: " OPENTELEMETRY_VERSION "}",
-      "{telemetry.sdk.name: opentelemetry}",
-      "{telemetry.sdk.language: cpp}",
-      "{res1: [1, 2, 3]}",
-      "  attributes    : {{attr1: [0, 1, 0]}}\n"
+      "  resource      : \n",
+      "service.name: unknown_service\n",
+      "telemetry.sdk.version: " OPENTELEMETRY_VERSION "\n",
+      "telemetry.sdk.name: opentelemetry\n",
+      "telemetry.sdk.language: cpp\n",
+      "res1: [1,2,3]\n",
+      "attributes    : \n",
+      "attr1: [0,1,0]\n"
```

Comment:
Should here be some alignment of `\t`?

Reply:
Yes, added alignment of `\t` to the expected output. The actual output does have alignment. The test doesn't fail as we do substring comparison.